frontend/src/packages/auth/auth-api.ts

```diff
@@ -32,8 +32,11 @@ class AuthApi extends BaseHttpApi {
         hasAuth: false,
       },
     );
+    const parsedResponse = await response.json<UserSignUpResponseDto>();
 
-    return await response.json<UserSignUpResponseDto>();
+    localStorage.setItem('token', parsedResponse.token);
```

Comment:
We already have a storage service as abstraction for localStorage, please reuse it

Reply:
And 'token' -> StorageKey.TOKEN

Reply:
> And 'token' -> StorageKey.TOKEN

That's also very important note, you wouldn't have thought about it, thank you!